src/Controller/Api/GroupController.php

```diff
@@ -55,15 +57,48 @@ public function __construct(GroupManagerInterface $groupManager, FormFactoryInte
     /**
      * Returns a paginated list of groups.
      *
-     * @ApiDoc(
-     *  resource=true,
-     *  output={"class"="Sonata\DatagridBundle\Pager\PagerInterface", "groups"={"sonata_api_read"}}
+     * @Operation(
+     *     tags={"/api/user/groups"},
+     *     summary="Returns a paginated list of groups.",
+     *     @SWG\Parameter(
+     *         name="page",
+     *         in="query",
+     *         description="Page for groups list pagination (1-indexed)",
+     *         required=false,
+     *         type="string"
+     *     ),
+     *     @SWG\Parameter(
+     *         name="count",
+     *         in="query",
+     *         description="Number of groups per page",
+     *         required=false,
+     *         type="string"
+     *     ),
+     *     @SWG\Parameter(
+     *         name="orderBy",
+     *         in="query",
+     *         description="Query groups order by clause (key is field, value is direction",
+     *         required=false,
+     *         type="string"
+     *     ),
+     *     @SWG\Parameter(
+     *         name="enabled",
+     *         in="query",
+     *         description="Enables or disables the groups only?",
```

Comment:
Why the question mark?

Reply:
It was copied from previus (legacy) files.

Reply:
Should we maybe challenge this? I mean I don't see a reason to have a question mark here, do you?

Reply:
IMO better will be do it in next RPs scope (fix swagger models).

Reply:
Okay, as you wish